### ISSUE TITLE
Publish latest release number as part of creating a final release

### ISSUE THREAD

**Issue by irbekrm** [cert-manager/release#42]
See https://github.com/cert-manager/website/pull/690#issuecomment-905434505 for context.

It would be good if, as part of cutting a new release that is meant to be the latest final (non-alpha/beta release)  we could also publish somewhere (probably a txt file in a bucket) the latest release version (i.e `v.1.5.3`) so we can then use that in our installation instructions (see i.e [kubectl install instructions](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux) where version number is read from [https://dl.k8s.io/release/stable.txt](https://dl.k8s.io/release/stable.txt)).

**Comment by irbekrm** [comment]
Thanks for investigating @maelvls 🙏🏼 

> Which bucket should that be?

It could have been nice if it was the same release bucket as that way we wouldn't need to make another bucket name configurable via params/would minimize the risk that someone accidentally overwrites the stable release name when testing publishing.
Perhaps we can make public just a single file in the release bucket publicly accessible?
Else I guess it'd have to be another bucket in the same project and we'd have to ensure that `cmrel` allows parameterizing that bucket name as well.

**Comment by maelvls** [comment]
You are correct, it is possible to set the `public-read` role to a single object, meaning that we can reuse the existing cert-manager-release bucket:

```
echo 1.5.3 >latest
gsutil cp latest gs://cert-manager-release
gsutil acl set public-read gs://cert-manager-release/latest
```

And then:

```sh
$ curl https://storage.googleapis.com/cert-manager-release/latest
1.5.3
```
